tests/system/providers/amazon/aws/example_sagemaker.py

```diff
@@ -421,6 +429,14 @@ def delete_logs(env_id):
     purge_logs(generated_logs)
 
 
+@task(trigger_rule=TriggerRule.ALL_DONE)
+def delete_model_group(group_name, model_version_arn):
+    sgmk_client = boto3.client("sagemaker")
+    # need to destroy model registered in group first
+    sgmk_client.delete_model_package(ModelPackageName=model_version_arn)
+    sgmk_client.delete_model_package_group(ModelPackageGroupName=group_name)
+
```

Comment:
Is there intention to create operator to do that?

Reply:
No plan yet. We might want to create this operator later but again, there is no plan as of today